data/pages-content/where.md

```diff
@@ -4,34 +4,47 @@ subtitle: EuroPython 2023 will be held in Prague!
 ---
 
 # Where
+<font size="+2.5">EuroPython 2023 will be held in **Prague**! We cannot wait to welcome you in one of the most beautiful cities in Europe!</font>
 
-EuroPython 2023 will be held in **Prague**! The capital of the Czech Repeblic
-and sitting on the [Vltava river](https://www.youtube.com/watch?v=l6kqu2mk-Kw), 
-Prague is one of the top five city destinations in Europe. The beautiful historic 
-centre is a UNESCO world heritage site. So, in addition to the conference, 
-there's a LOT to see, do and enjoy. Prague has an extensive and modern public 
-transport system, so travelling around won't be a problem and getting there is 
-easy via well serviced train, air and road infrastructure. With an average 
-July high of 25°c, remember to pack your shorts and sandals with your laptop, 
-along with a spirit of adventure to explore one of the most visited and vibrant 
-cities in Europe.
+## Prague
+<img src="/img/Prague.jpeg" width="3279" height="1785"/>
```

Comment:
Will the width and height work on mobile..?

Reply:
it does but I didn't need to put it there. So deleted!